src/pages/components/PageDetailsPage/PageDetailsPage.tsx

```diff
@@ -208,13 +223,14 @@ const PageDetailsPage: React.FC<PageDetailsPageProps> = ({
               attributeValues={getAttributeValuesFromReferences(
                 assignReferencesAttributeId,
                 data.attributes,
-                referencePages
+                referencePages,
```

Comment:
I know this isn't a change made in this PR but the rest of the project opens modal dialogs using query strings and keep them outside page components. I intend to keep it this way.